Gulpfile.mjs

```diff
@@ -260,6 +260,7 @@ function createWorker(useWorker) {
     return require("./babel-worker.cjs");
   }
   const worker = new JestWorker(require.resolve("./babel-worker.cjs"), {
+    enableWorkerThreads: true,
```

Comment:
This is 0.3s faster on my local machine. :)

Reply:
Unfortunately it will be 1s slower on mac. But anyway it is not really noticeable.

```
# enableWorkerThreads: true
$ make clean-lib && time yarn gulp build-babel
yarn gulp build-babel  11.68s user 1.15s system 427% cpu 3.001 total

# enableWorkerThreads: false
$ make clean-lib && time yarn gulp build-babel
yarn gulp build-babel  10.75s user 1.10s system 404% cpu 2.930 total
```